### PR TITLE
Add User Stories and Update Use Cases

### LINKED ISSUE #135: [Eugene] Add User Stories to DG
Add User stories based on what was assigned previously. Can refer to [document](https://docs.google.com/document/d/1kDJEZUjy2-hNoUon0OQ7hTY5kcZyooB7Hj3U_QrR4r8/edit?pli=1#)

### PR DESCRIPTION
**Pull request by Eugenetanwl3881**
Closes #135 

Added User Stories and formatted use cases for standardization of terms used.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -395,8 +409,8 @@ MSS:
    Extensions:
 
 - 1a. Item does not exist
-  1a1. Foodrem displays an error.
-  1a2. Foodrem asks the user if they want to try again
+  1a1. FoodRem displays an error.
+  1a2. FoodRem asks the user if they want to try again
```
Comment (bryanljx) [inline docs/DeveloperGuide.md:413]: Missing line breaks. Probably better to standardize and have each step on a new line instead. 

Current it looks something like this:
```
- 1a. Item does not exist 1a1. FoodRem displays an error. 1a2. FoodRem asks the user if they want to try again
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -267,15 +267,29 @@ Inventory managers who are proficient with typing for small F&B businesses
...
 
-Updating inventory items
+Delete Inventory Item:
+
+Update inventory items:
 1. As a purchasing manager, I can rename my inventory items, so that I can update items with an incorrect name.
 2. As a purchasing manager, I can set the inventory quantity of my items.
 3. As a purchasing manager, I can set the date I bought my inventory, so that I know how long I have been keeping certain items..
 4. As a purchasing manager, I can set the date my inventory will expire, so that I know when certain items need to be consumed or used. 
 5. As a purchasing manager, I can increase the inventory quantity of my items, so that I can keep my stock updated when I purchase new items..
```
Comment (bryanljx) [inline docs/DeveloperGuide.md:279]: Is there a reason why some lines end with 2 full stops and some end with 1?

**Review by bryanljx — CHANGES_REQUESTED** [review]
Looks good overall. Minor issues to fix before merging.

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (Eugenetanwl3881) [reply docs/DeveloperGuide.md]: Will remove

**Review by bryanljx — APPROVED** [review]

**Pull request merged**